static/app/components/replays/replayController.tsx

```diff
@@ -60,27 +62,26 @@ function ReplayCurrentTime() {
 
 function ReplayPlaybackSpeed({speedOptions}: {speedOptions: number[]}) {
   const {setSpeed, speed} = useReplayContext();
-
   return (
-    <ButtonBar active={String(speed)} merged>
-      {speedOptions.map(opt => (
-        <Button
-          key={opt}
-          size="xsmall"
-          barId={String(opt)}
-          onClick={() => setSpeed(opt)}
-          title={t('Set playback speed to %s', `${opt}x`)}
-        >
-          {opt}x
-        </Button>
-      ))}
-    </ButtonBar>
+    <StyledOptionSelector
+      title={`${t('Speed')}`}
+      selected={`${speed}`}
+      options={speedOptions.map(speedOption => ({
+        value: `${speedOption}`,
+        label: `${speedOption}x`,
+        disabled: speedOption === speed,
+      }))}
+      onChange={(opt: string) => {
+        setSpeed(parseInt(opt, 10));
+      }}
+      containerPadding={4}
+    />
```

Comment:
Try as I might I couldn't make the dropdown smaller. Removing the border would solve the problem since everything is horizontally aligned. Just let me know what needs to change.